runelite-client/src/main/java/net/runelite/client/plugins/timersandbuffs/GameTimer.java

```diff
@@ -97,8 +97,8 @@ enum GameTimer
 	SILK_DRESSING(ItemID.SILK_DRESSING_2, GameTimerImageType.ITEM, "Silk dressing", 100, GAME_TICKS, true),
 	BLESSED_CRYSTAL_SCARAB(ItemID.BLESSED_CRYSTAL_SCARAB_2, GameTimerImageType.ITEM, "Blessed crystal scarab", 40, GAME_TICKS, true),
 	SPELLBOOK_SWAP(SpriteID.SPELL_SPELLBOOK_SWAP, GameTimerImageType.SPRITE, "Spellbook Reset", 120, ChronoUnit.SECONDS, false),
-	GOADING(ItemID.GOADING_POTION4, GameTimerImageType.ITEM, "Goading potion", 6, ChronoUnit.MINUTES, true),
-	PRAYER_REGENERATION(ItemID.PRAYER_REGENERATION_POTION4, GameTimerImageType.ITEM, "Prayer regeneration", 8, ChronoUnit.MINUTES, true),
+	GOADING(ItemID.GOADING_POTION4, GameTimerImageType.ITEM, "Goading potion", true),
+	PRAYER_REGENERATION(ItemID.PRAYER_REGENERATION_POTION4, GameTimerImageType.ITEM, "Prayer regeneration", true),
```

Comment:
These should be changed to have `removeOnDeath` be false, as they're entirely controlled by var updates. If dying removes these timers, they would be removed as a result of vars being set to 0 after dying.

Reply:
My bad, that makes a lot of sense. I assumed Adam had tested it and for some reason the varbits would incorrectly persist while the effect ended at death. Turns out, the prayer regeneration even persists after death! <https://youtu.be/i50oLo3WN4o>